docs/feature/policies/workflow-step-policy.md

```diff
@@ -0,0 +1,27 @@
+---
+sidebar_position: 4
+---
+# Workflow step policy
+
+The Workflow step policy, is an optional policy, which can be provided in the Implementation manifest in an Action step. This policy can be used when:
```

Comment:
I would add information that this can be added only by the content creator when creating a Manifest. And cannot be changed later. 
It can be overwritten when the default order is used + link to the merge policices section